source/nodejs/adaptivecards-site/themes/adaptivecards/layout/_partial/section-details-item.ejs

```diff
@@ -8,7 +8,7 @@
              <% if (typeof smallImageAlt !== 'undefined') { %>
              alt="<%- smallImageAlt %>"
              <% } else { %>
-             aria-role="presentation"
+             role="none"
```

Comment:
Can you reword title and description of the PR? `Add alt text to images on home page ` doesn't match with code

Reply:
Good call! The fix changed and I forgot to change the PR title